### PR TITLE
News entry on tetrahedral remeshing

### PR DESCRIPTION
**Pull request by janetournois**
1st version of a news entry on tetrahedral remeshing

### CODE REVIEW

File: _posts/2020-08-07-Tetrahedral-remeshing.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+<h3>Tetrahedral Isotropic Remeshing</h3>
+
+<p>This new package provides an algorithm for quality tetrahedral remeshing. This practical iterative remeshing algorithm
+is designed to remesh multi-material tetrahedral meshes, by iteratively performing a sequence of elementary operations
+such as edge splits, edge collapses, edge flips, and vertex relocations following a Laplacian smoothing.
+The algorithm results in high-quality uniform isotropic meshes, with the desired mesh density, while preserving
+the input geometric curve and surface features.<p>
+
+(see <a href="https://cgal.geometryfactory.com/CGAL/doc/master/Tetrahedral_remeshing/group__PkgTetrahedralRemeshingRef.html#gae4686577872290e0335439edafb27a7b">CGAL::tetrahedral_isotropic_remeshing()</a>)
```
Comment (sloriot) [inline _posts/2020-08-07-Tetrahedral-remeshing.md:26]: I would remove that

File: _posts/2020-08-07-Tetrahedral-remeshing.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+<br>
+
+<br>
+<div style="text-align:center;">
+  <a href="../../../../images/Tet_remeshing_star.png"><img src="../../../../images/Tet_remeshing_star.png" style="max-width:45%"/></a>
```
Comment (sloriot) [inline _posts/2020-08-07-Tetrahedral-remeshing.md:32]: the rendering is strange. Did you try not imposing any width?

**Comment by lrineau** [comment]
From what I see in upcoming.cgal.org, the picture and its caption render strangely, on my laptop:

Screenshot_20200817_113527

**Comment by janetournois** [comment]
Right, thank you

**Review by sloriot — APPROVED** [review]

**Comment by sloriot** [comment]
@janetournois feel free to merge it if you no longer want to change something.

**Pull request merged**